migration/idempotent/001-base.sql

```diff
@@ -2775,11 +2775,65 @@ BEGIN
         RETURN;
     END IF;
 
+    IF NOT _prom_catalog.is_multinode() THEN
+        RETURN QUERY
+        SELECT
+            m.id,
+            m.metric_name,
+            m.table_name::text as table_name,
+            ARRAY
+            (
+                SELECT lkp.key
+                FROM _prom_catalog.label_key_position lkp
+                WHERE lkp.metric_name = m.metric_name
+                ORDER BY lkp.key
+            ) label_keys,
+            coalesce(m.retention_period, _prom_catalog.get_default_retention_period()) as retention_period,
+            _timescaledb_internal.to_interval(d.interval_length) as chunk_interval,
+            x.compressed_interval,
+            x.total_interval,
+            x.before_compression_bytes,
```

Comment:
This used to be called `before_compression_total_bytes`, and the line below used to be called `after_compression_total_bytes`.

Reply:
Just to clarify: this is the same as the "no timescaledb" branch, but different to the "multinode" branch.

Reply:
the function's signature determines the actual names and it does not include `total`. I'll change the multinode branch